app/src/controllers/object.js

```diff
@@ -691,6 +687,7 @@ const controller = {
       const metadata = getMetadata(req.headers);
       const tagging = req.query.tagset;
       const params = {
+        userId: req.query.userId,
```

Comment:
This userId attribute should be declared after the bucketId attribute.

Also, if we're going to be adding userId as a queryable parameter, we're going to need to make sure that we are handling not only the singular case, but also the plural case at some point. This should have mixedQueryToArray, but IIRC we don't actually support that atm in the filterUserId modifier on the service/db layers though, so this may need to be dealt with as a future PR.